cirq/sim/mux.py

```diff
@@ -52,6 +52,53 @@ def sample(program: Union[circuits.Circuit, schedules.Schedule],
             repetitions=repetitions)
 
 
+def out_vector(program: Union[ops.Operation, ops.Gate, circuits.Circuit, schedules.Schedule],
```

Comment:
What's the motivation for calling it an `out_vector`?

Reply:
wonder if we could make this more of a verb?

Reply:
This shouldn't be called "get out vector" because I want to be consistent with `cirq.unitary`, which also avoids using a verb. We are asking for a property.

Renamed to `final_wavefunction`.